lib/mayaHydra/hydraExtensions/mayaUtils.h

```diff
@@ -176,6 +176,15 @@ bool SetNodeAttribute(MObject node, std::string attrName, AttrType newValue)
  */
 MObject GetShadingGroupFromShader(const MObject& shader);
 
+/**
+ * @brief Get if this MDagPath is an Arnold sky dome light.
+ *
+ * @param[in] dagPath is a MDagPath
+ *
+ * @return true if the object is a sky dome light, false otherwise
+ */
+bool IsDagPathAnArnoldSkyDomeLight(const MDagPath& dagPath);
```

Comment:
This makes me uncomfortable.  Why are we treating Arnold lights in a special way, with a special conditional?  Is there no way that our adapter infrastructure can handle this?  What if another renderer has a different way of handling dome lights?

Reply:
This is specific to maya hydra, not specific to a renderer.